playwright/helper.py

```diff
@@ -105,9 +115,10 @@ class Header(TypedDict):
 
 
 class ContinueParameters(TypedDict, total=False):
-    method: str
-    headers: List[Header]
-    postData: str
+    url: Optional[str]
```

Comment:
since total=False all keys inside the dict are optional, so  afaik you don't have to specify it.

Reply:
I'd like to treat it as a type, probably total=False should not be there

Reply:
I think mypy then complains from the user perspective if you only specify a single property.